.eslintrc.js

```diff
@@ -11,15 +11,20 @@ module.exports = {
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "prettier/@typescript-eslint",
+    "prettier",
   ],
   rules: {
     ...jsConfig.rules,
+    "@typescript-eslint/no-unsafe-argument": "off",
     "@typescript-eslint/array-type": "error",
     "@typescript-eslint/consistent-type-assertions": "error",
     "@typescript-eslint/consistent-type-definitions": "error",
+    "@typescript-eslint/no-redundant-type-constituents": "error",
```

Comment:
New rule that is on by default it seems, so I added it to be explicit as it seems useful.